src/index.html

```diff
@@ -34,6 +34,55 @@ <h1>FitLit</h1>
     </ul>
   </header>
   <main>
+    <section class="main__section--hydration main__sections">
+      <h3>
+        <button class="section__btn--activity">Activity</button>
+        <button class="hydration__btn--hydration">Hydration</button>
+        <button class="sleep__btn--sleep">Sleep</button>
+      </h3>
+      <article class="addActivity__article">
+        <form method="POST" hidden>
+          <label>Date</label>
+          <p class="form__activity--error" hidden>Enter a Valid Date</p>
+          <input aria-required="true" class="form-control invalid" type="text" name="date" pattern="" placeholder="YYYY/DD/MM">
+          <label>Number of Steps</label>
+          <p class="form__activity--error" hidden>Enter a Valid Number</p>
+          <input aria-required="true" class="form-control invalid" type="number" name="number" pattern="" min="1" max="5" placeholder="####">
+          <label>Minutes Active</label>
+          <p class="form__activity--error" hidden>Enter a Valid Number</p>
+          <input aria-required="true" class="form-control invalid" type="number" name="number" pattern="" min="1" max="5" placeholder="####">
+          <label>Flights of Stairs</label>
+          <p class="form__activity--error" hidden>Enter a Valid Number</p>
+          <input aria-required="true" class="form-control invalid" type="number" name="number" pattern="" min="1" max="5" placeholder="####">
+          <button type="submit">Submit</button>
+        </form>
+      </article>
+      <article class="addHydration__article">
+        <form method="POST">
+            <label>Date</label>
+            <p class="form__hydration--error" hidden>Enter a Valid Date</p>
+            <input aria-required="true" class="form-control invalid" type="text" name="date" pattern="" placeholder="YYYY/DD/MM">
+            <label>Ounces of Water Drank</label>
+            <p class="form__hydration--error" hidden>Enter a Valid Number</p>
+            <input aria-required="true" class="form-control invalid" type="number" name="number" pattern="" min="1" max="2" placeholder="##">
+            <button type="submit">Submit</button>
+          </form>
+      </article>
+      <article class="addSleep__article">
+        <form method="POST" hidden>
```

Comment:
I know I mentioned this when you showed us earlier, but the accessibility on the forms is fantastic.  Nice work!